rte_ctrl.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+PFX="api/v1"
+
+CMD_ID_RELAY=0
+CMD_ID_RESET=8
+CMD_ID_POWER=9
+
+# TODO add host address as positional argument
+HOST=http://192.168.4.38:8000
```

Comment:
```suggestion
RTE_IP="${RTE_IP:-127.0.0.1}"
HOST="http://${RTE_IP}:8000"
```

Reply:
This will let you override IP via env variable, so script can be used both on RTE and host

Reply:
The problem is, curl uses an ip address rather than a string, so this solution doesn't work. I've tried to parse the string to an address, to no avail.
`HOST=http://192.168.4.43:8000`

Reply:
@Silentsky0 It works. I do not know what you mean. There is no such think as "IP address" type. It is alwaays a string in bash. See here: https://github.com/3mdeb/RteCtrl/pull/8

Reply:
Solved